src/library/scala/collection/SortedOps.scala

```diff
@@ -12,6 +12,11 @@ trait SortedOps[A, +C] {
   /** Returns the last key of the collection. */
   def lastKey: A
 
+  /** Comparison function that orders keys. */
+  @deprecated("Use Ordering.compare instead.", "2.13.0")
```

Comment:
Actually the suggestion is not correct, it should be `ordering.compare` instead (lower case).

Reply:
updated. thanks